tests/v2/test_wakunode.nim

```diff
@@ -109,3 +109,37 @@ procSuite "WakuNode":
     check:
       (await completionFut.withTimeout(5.seconds)) == true
     await allFutures([node1.stop(), node2.stop()])
+
+  asyncTest "Store protocol returns expected message":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.init(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        Port(60000))
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node2 = WakuNode.init(nodeKey2, ValidIpAddress.init("0.0.0.0"),
+        Port(60002))
+      contentTopic = "foobar"
+      message = WakuMessage(payload: "hello world".toBytes(), contentTopic: contentTopic)
+      uuid = "123"
```

Comment:
I suppose this is fine as a test, but we should really call a genUUID function here and make it explicit that this is a request id.

Reply:
https://github.com/status-im/nim-waku/issues/183